shoop/xtheme/parsing.py

```diff
@@ -184,25 +186,33 @@ def parse(self, parser):
         :rtype: jinja2.nodes.Output
         """
         lineno = next(parser.stream).lineno
+        global_type = False
+        placeholder_real_name = parser.stream.current.value
+        if parser.stream.look().value == "global":
+            global_type = True
+            next(parser.stream)
         placeholder_name = six.text_type(parse_constantlike(self.environment, parser))
-        self._new_layout(parser, placeholder_name)
+        placeholder_name = placeholder_real_name
```

Comment:
What is happening here? You are always assigning parser.stream.current.value into placeholder_real_name.

Could this be like:
```
global_type = bool(parser.stream.look().value == "global")
if global_type:
    placeholder_name = parser.stream.current.value
    next(parser.stream)
else: 
    placeholder_name = six.text_type(parse_constantlike(self.environment, parser))
```